### PR TITLE
docs: update global-rule/plugin-config/plugin/ docs

### PR DESCRIPTION
**Pull request by hf400159**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/terminology/plugin-config.md [hunk 1]
```diff
@@ -26,45 +26,60 @@ description: Plugin Config in Apache APISIX.
...
 
-For example, if we configure a Plugin Config as shown below
+If a Route already has the `plugins` field configured, the plugins in the Plugin Config will effectively be merged to it.
 
-```
+The same plugin in the Plugin Config will not override the ones configured directly in the Route. For more information, see [Plugin](./plugin.md).
+
+:::
+
+For example, if you configure a Plugin Config as shown below
```
Comment (SylviaBABY) [inline docs/en/latest/terminology/plugin-config.md:78]:
```suggestion
For example, if you configure a Plugin Config as shown below:
```

File: docs/zh/latest/terminology/plugin.md [hunk 2]
```diff
@@ -21,15 +27,29 @@ title: Plugin
...
 
-在 `conf/config.yaml` 中，可以声明本地 APISIX 节点都支持哪些插件。这是个白名单机制，不在该白名单的插件配置，都将会被自动忽略。这个特性可用于临时关闭或打开特定插件，应对突发情况非常有效。
-如果你想在现有插件的基础上新增插件，注意需要拷贝 `conf/config-default.yaml` 的插件节点内容到 `conf/config.yaml` 的插件节点中。
+你也可以参考 [Admin API](../admin-api.md#plugin) 了解如何使用该资源。
 
-一个插件在一次请求中只会执行一次，即使被同时绑定到多个不同对象中（比如 Route 或 Service）。
-插件运行先后顺序是根据插件自身的优先级来决定的，例如：
+:::note 注意
+
+对于同一个插件的配置，只能有一个是有效的，优先级为 Consumer > Route > Plugin Config > Service。
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/plugin.md:38]:
```suggestion
对于同一个插件的配置，只能有一个是有效的，其插件配置优先级为 Consumer > Route > Plugin Config > Service。
```

File: docs/zh/latest/terminology/plugin.md [hunk 3]
```diff
@@ -21,15 +27,29 @@ title: Plugin
...
+
+:::
+
+## 配置简介
+
+如果你想在现有插件的基础上新增插件，请复制 `./conf/config-default.yaml` 中的 `plugins` 参数下的插件列表到 `./conf/config.yaml` 的 `plugins` 参数中。
+
+:::tip 提示
+
+在 `./conf/config.yaml` 中的 `plugins` 参数中，可以声明本地 APISIX 节点支持了哪些插件。这是个白名单机制，不在该白名单的插件配置，都将会被自动忽略。该特性可用于临时关闭或打开特定插件，应对突发情况非常有效。
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/plugin.md:48]:
```suggestion
在 `./conf/config.yaml` 中的 `plugins` 参数中，可以声明本地 APISIX 节点支持了哪些插件。这是个白名单机制，不在该白名单的插件配置将被自动忽略。该特性可用于临时关闭或打开特定插件，应对突发情况非常有效。
```

File: docs/zh/latest/terminology/plugin.md [hunk 4]
```diff
@@ -60,24 +77,30 @@ local _M = {
...
+| 名称           | 类型           |     描述       |
+|--------------- |-------------- |----------------|
+| disable        | boolean       | 当设置为 `true` 时，则禁用该插件。可选值为 `true` 和 `false`。 |
+| error_response | string/object | 自定义错误响应。 |
+| priority       | integer       | 自定义插件优先级。 |
+| filter         | array         | 根据请求的参数，在运行时控制插件是否执行。此配置由一个或多个 {var, operator, val} 元素组成列表，类似：`{{var, operator, val}, {var, operator, val}, ...}}`。例如 `{"arg_version", "==", "v2"}`，表示当前请求参数 `version` 是 `v2`。这里的 `var` 与 NGINX 内部自身变量命名是保持一致。操作符的使用方法，请参考 [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list)。|
+
+### 禁用指定插件
+
+通过 `disable` 参数，你可以新增一个处于禁用状态的插件，请求不会经过该插件。
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/plugin.md:103]:
```suggestion
通过 `disable` 参数，你可以将某个插件调整为“禁用状态”，即请求不会经过该插件。
```

**Review by SylviaBABY — APPROVED** [review]

File: docs/en/latest/terminology/plugin-config.md [hunk 5]
```diff
@@ -26,45 +26,60 @@ description: Plugin Config in Apache APISIX.
...
-```shell
-curl http://127.0.0.1:9180/apisix/admin/plugin_configs/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -i -d '
-{
-    "desc": "blah",
-    "plugins": {
-        "limit-count": {
-            "count": 2,
-            "time_window": 60,
-            "rejected_code": 503
+    ```shell
```
Comment (juzhiyuan) [inline docs/en/latest/terminology/plugin-config.md:39]: Why add some spaces here? 😄

File: docs/zh/latest/terminology/plugin.md [hunk 6]
```diff
@@ -41,13 +61,10 @@ local _M = {
...
 ```
 
-插件配置可存放于 Route 或 Service 中，键为 `plugins`，值是包含多个插件配置的对象。对象内部用插件名字作为 key 来保存不同插件的配置项。
-
-如下所示的配置中，包含 `limit-count` 和 `prometheus` 两种插件的配置：
+插件的配置信息，可以存放 Route、Service、Plugin Config 等对象中的 `plugins` 参数下。如下所示的配置中，包含 `limit-count` 和 `prometheus` 两个插件的配置信息：
 
 ```json
 {
-    ...
```
Comment (juzhiyuan) [inline docs/zh/latest/terminology/plugin.md:50]: 🤔 I'm not sure if we should keep this part, as it's inside the Route Object.

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**